builtin/credential/cert/path_certs.go

```diff
@@ -228,6 +234,23 @@ func (b *backend) pathCertWrite(ctx context.Context, req *logical.Request, d *fr
 		}
 	}
 
+	var parsedCIDRs []*sockaddr.SockAddrMarshaler
+	if boundCIDRListRaw, ok := d.GetOk("bound_cidrs"); ok {
+
+		var boundCIDRList []string
+		if boundCIDRs, ok := boundCIDRListRaw.([]string); ok {
```

Comment:
There is a zero value for TypeCommaStringSlice of nil slice and this cast checking is not required.

Reply:
:taco: